tests/unit/test_vcs.py

```diff
@@ -366,3 +366,9 @@ def test_subversion__get_url_rev_options():
 def test_get_git_version():
     git_version = Git().get_git_version()
     assert git_version >= parse_version('1.0.0')
+
+
+@pytest.mark.svn
+def test_get_vcs_version():
```

Comment:
Since this is testing a method of the `Subversion` class, it would be better to call this  `test_subversion__get_vcs_version()` (like one of the tests above).

Also, in order to test the parsing more deterministically, can you also add a separate test function that patches the return value of `run_command()`? You should be able to find some examples in the test suite. This can be a parametrized test with at least two test cases: a normal case and the "unparseable" case. These tests should check the return value using equality rather than inequality. And since it won't require SVN to be installed, you won't need the svn decorator.

Reply:
> You should be able to find some examples in the test suite. 

I'm having trouble finding an example. Do you know of one I could look at?

Reply:
Here is one using both a mock return value and test parametrization: https://github.com/pypa/pip/blob/54b6a91405adc79cdb8a2954e9614d6860799ccb/tests/unit/test_vcs.py#L171-L185

I would give the argument the name `expected` though rather than `result`.